apis/python/src/tiledbsoma/dataframe.py

```diff
@@ -288,22 +262,17 @@ def write(self, values: pa.Table) -> None:
                     )
                 )
 
-        if self._get_is_sparse():
-            # sparse write
-            with self._tiledb_open("w") as A:
-                A[rowids] = attr_cols_map
-        else:
-            # TODO: This was a quick thing to bootstrap some early ingestion tests but needs more thought.
-            # In particular, rowids needn't be either zero-up or contiguous.
-            assert len(rowids) > 0
-            rowids = sorted(rowids)
-            assert rowids[0] == 0
-
-            # dense write
-            lo = rowids[0]
-            hi = rowids[-1]
-            with self._tiledb_open("w") as A:
-                A[lo : (hi + 1)] = attr_cols_map
+        # TODO: This was a quick thing to bootstrap some early ingestion tests but needs more thought.
+        # In particular, rowids needn't be either zero-up or contiguous.
```

Comment:
this section looks wrong to me - several issues:
* we need to change the API for write to take a single dimension sub-array coordinates (like DenseNdArray, but just a single dimension),  after which all of this stuff can be removed
* and there must be a test to confirm that the table does not contain the soma_rowid column, as that would be in conflict with the function param specifying the sub-array coordaintes

Reply:
A simple unit test will test this - ie,
* create a DataFrame
* write a single row at position 0
* write 10 rows at position 1-11
* etc.